### DISCUSSION TITLE
Where's the Exe for PCBox?

### DISCUSSION THREAD

**Discussion by Olddantrucker** [PCBox/PCBox#33] [Q&A] [answered]
Hi. Just saw this stumbling by and downloaded but don't see an EXE or any kind of installer to setup.  How do you even start it?  I just get a bunch of gibberish files. Well gibberish to me.   https://github.com/PCBox/PCBox/discussions/new

**Comment by bachimusprime** [comment, marked as answer]
Click on the Actions tab, and click the latest MinGW Makefiles job. Scroll to the bottom to where it says Artifacts and download a build. In the future, please head to our Discord server or IRC channel for support.

**Reply by bachimusprime** [reply]
Yes there is dumbass, look closer. Also, I told people to click the latest build that is a MinGW Makefiles build. It's also not dead, dumbass. This project has been updated many times over the past few weeks. Look through the commit logs, and it's obvious. Stop assuming shit when you clearly know fucking nothing.

**Reply by bachimusprime** [reply]
Also I'm a girl, and ALSO, 86Box is another option and is even referenced in the About section of the repository. PCBox is even a fork of 86Box and regularly keeps up to date with it. Learn how to fucking read, jackass.

**Reply by Burinis** [reply]
I have a question for you: why is it that all PCBox builds are only in the Actions tab, and not in the Releases column, as is the case with all other author's utilities on Github?

**Reply by bachimusprime** [reply]
@Burinis Simple, PCBox is currently a rolling-release emulator, with no stable releases. This is not only because I want to ensure that when there is a stable release, all the major features that PCBox has over 86Box work properly, but also because I don't want people to use old outdated code just because it's marked as a "stable release" when the latest code is almost always better. Hence, I have GitHub automatically build PCBox on every new commit, and it then publishes the result of that build as an "artifact" if the build succeeds. Ideally you should be updating PCBox as often as possible, to ensure that you not only have the best experience with PCBox, but also to ensure that the GitHub repo's issues don't get bugs reported that have already been fixed in the latest code. This is an issue a lot of other emulators have, where unsuspecting users download the latest stable release, which is almost always at least somewhat outdated, and then proceed to report bugs that have already been fixed in the latest code. Even 86Box has this happen sometimes. If you're using an emulator, you should ALWAYS use the absolute latest code you can get. It is almost always better in one way or another.

**Comment by bca009** [comment]
I would like to know that too Sent from my iPadOn Aug 30, 2023, at 10:47 AM, Burinis ***@***.***> wrote:﻿
I have a question for you: why is it that all PCBox builds are only in the Actions tab, and not in the Releases column, as is the case with all other author's utilities on Github?

**Reply by bachimusprime** [reply]
Simple, PCBox is currently a rolling-release emulator, with no stable releases. This is not only because I want to ensure that when there is a stable release, all the major features that PCBox has over 86Box work properly, but also because I don't want people to use old outdated code just because it's marked as a "stable release" when the latest code is almost always better. Hence, I have GitHub automatically build PCBox on every new commit, and it then publishes the result of that build as an "artifact" if the build succeeds. Ideally you should be updating PCBox as often as possible, to ensure that you not only have the best experience with PCBox, but also to ensure that the GitHub repo's issues don't get bugs reported that have already been fixed in the latest code. This is an issue a lot of other emulators have, where unsuspecting users download the latest stable release, which is almost always at least somewhat outdated, and then proceed to report bugs that have already been fixed in the latest code. Even 86Box has this happen sometimes. If you're using an emulator, you should ALWAYS use the absolute latest code you can get. It is almost always better in one way or another.
